js/common/modules/@arangodb/graph/graphs-generation.js

```diff
@@ -0,0 +1,179 @@
+/*jshint globalstrict:false, strict:false */
+/*global assertEqual, assertTrue, assertNotEqual, JSON */
+'use strict';
+
+// //////////////////////////////////////////////////////////////////////////////
+// / @brief Pregel Tests: graph generation
+// /
+// / @file
+// /
+// / DISCLAIMER
+// /
+// / Copyright 2022 ArangoDB GmbH, Cologne, Germany
+// /
+// / Licensed under the Apache License, Version 2.0 (the "License")
+// / you may not use this file except in compliance with the License.
+// / You may obtain a copy of the License at
+// /
+// /     http://www.apache.org/licenses/LICENSE-2.0
+// /
+// / Unless required by applicable law or agreed to in writing, software
+// / distributed under the License is distributed on an "AS IS" BASIS,
+// / WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// / See the License for the specific language governing permissions and
+// / limitations under the License.
+// /
+// / Copyright holder is ArangoDB GmbH, Cologne, Germany
+// /
+// / @author Roman Rabinovich
+// / @author Copyright 2022, ArangoDB GmbH, Cologne, Germany
+// //////////////////////////////////////////////////////////////////////////////
+
+const makeEdge = (from, to, vColl, name_prefix, inverse = false) => {
```

Comment:
More contra: a boolean parameter does not make it clear what `makeEdge(v, w, true)` supposed to do; 

You will be computing the value of that parameter either way in the code that calls `makeEdge`, and there it will (hopefully) be obvious what it does.

Reply:
Removed the parameter.